src/test/QuotePage.test.js

```diff
@@ -0,0 +1,18 @@
+import { render, screen } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import QuotePage from '../pages/QuotePage';
+/* eslint-disable no-unused-expressions */
+
```

Comment:
- [ ] Please do not disable linters because it helps us to write better and clean codes. Kindly fix the linter error instead of disabling it.

Reply:
Resolved 👍🏽